Priyaa/HTML Basics Remake/Assignment-3.html

```diff
@@ -7,7 +7,7 @@
 
 <body>
     <h1><b>Register</b></h1>
-    <form id="registration_form">
+    <form id="registration_form"  method="get" autocomplete="on">
```

Comment:
The form response would be `post` and not `get`. Refer this: [HTTP Request Methods - w3schools](https://www.w3schools.com/tags/ref_httpmethods.asp)

And also, don't use `on`. Use specific ones for each input. Refer this: [The HTML autocomplete attribute - MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)